documentation/docs/assertions/collections.md

```diff
@@ -0,0 +1,50 @@
+---
+id: collections
+title: Collection Matchers
+slug: collection-matchers.html
+sidebar_label: Collections
+---
+
+This page describes the rich assertions (matchers) that are available for Collection, Iterable and Array types.
+
```

Comment:
Would be worthwhile to mention inspectors here as well IMO

Reply:
done